packages/adapter-squid/src/validation/StepConfigValidator.ts

```diff
@@ -10,34 +10,34 @@ import {
    IValidator,
    ValidationError,
 } from "@mate/sdk";
-import { BridgeSwapStepCodec, SquidStep } from "../types";
+import { BridgeSwapConfigCodec, SquidStepConfig } from "../types";
 import { ISquidProvider, SquidProvider } from "../services";
 import { isAddress } from "viem";
 
 @injectable()
-export class StepConfigValidator implements IValidator<SquidStep> {
+export class StepConfigValidator implements IValidator<SquidStepConfig> {
    constructor(@inject(SquidProvider) private squidProvider: ISquidProvider) {}
 
-   public async validate(data: any): Promise<SquidStep> {
```

Comment:
would suggest restricting explicit any in the config

```suggestion
   public async validate(data: unknown): Promise<SquidStepConfig> {
```